src/components/TokenTotalLineItem.tsx

```diff
@@ -12,7 +12,7 @@ import { LocalAmount } from 'src/transactions/types'
 
 interface Props {
   tokenAmount: BigNumber
-  tokenAddress: string
+  tokenAddress?: string
```

Comment:
should this be string | null? since blockchain API returns null and doesn't drop the field?